docs/administrator-documentation/moderne-dx/how-to-guides/dx-org-service.md

```diff
@@ -0,0 +1,77 @@
+---
+sidebar_label: Creating an organizations service
+description: How to create and configure an organizations service that pairs with Moderne DX.
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Creating an organizations service
+
+You should create a dedicated organizations service if you want to:
+
+* Limit access to the organizations you've [previously defined](./configure-dx-organizations.md) so that some users only have access to some repositories OR
+* Customize commit messages by repository (e.g., adding a JIRA ticket to your commit messages based on the repository)
+
+This guide will walk you through everything you need to know to create such a service.
+
+#### Prerequisites
+
+This guide assumes that:
+
+* You are an admin of Moderne DX
+* You have deployed Moderne DX in your environment
+* You've already [created an organizational hierarchy for your repositories](./configure-dx-organizations.md)
+
+## Organizations service template and API
+
+You have two main options for building this service. You can:
+
+1. (**Recommended**) Fork our [Organizations service template](https://github.com/moderneinc/moderne-organizations) and modify it to meet your needs. Please see the [README](https://github.com/moderneinc/moderne-organizations/blob/main/README.md) for how to spin this up quickly. It can be as simple as updating a CSV file.
+2. Build your own service that fulfills the [GraphQL contract](https://github.com/moderneinc/moderne-organizations/blob/main/src/main/resources/schema/organizations.graphqls) using any GraphQL stack (e.g., NodeJS, Rust, C#, etc.)
```

Comment:
Making a note here to remind myself on Monday that we need to update the template to remove all the extra stuff before this goes out.